dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentity.Tests/UnitTests.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.Extensions.Configuration;
+using ReturnInsertedIdentityWithDapper;
+using static ReturnInsertedIdentityWithDapper.Methods;
+
+namespace ReturnInsertedIdentity.Tests;
+
+public class Tests
+{
+    private string _connectionString;
+
+    [SetUp]
+    public void Setup()
+    {
+        _connectionString = new ConfigurationBuilder()
+            .SetBasePath(AppContext.BaseDirectory)
+            .AddJsonFile("appsettings.json")
+            .Build()
+            .GetConnectionString("DefaultConnection")!;
+    }
+
+    [Test]
+    public void GivenOutputInsertedMethod_WhenNewRecord_ThenReturnedIDGreaterThenZeroLiveTest()
```

Comment:
`Then` is spelled incorrectly, it should be `Than`

Reply:
Fixed typo.